data-es6.js

```diff
@@ -1689,6 +1691,7 @@ exports.tests = [
         chrome49:    true,
         webkit:      true,
         edge12:      flag,
+		edge14:      true,
```

Comment:
there must be mixed tab/spaces, as I copy pasted from the ES6 file to make sure…I'll update.